Bottom: 1ef4dab5ed00c7c176235807ae7ce75570d1c630
Top:    74b33931ed28c46a5dd52a1deff5c181eee58725
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 18:11:28 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/lib/master.c b/lib/master.c
index e105e63..3f79f61 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1552,10 +1552,9 @@ next:
 			count++;
 
 			source = source->next;
-
-			lookup_close_lookup(ap);
 		}
 
+		lookup_close_lookup(ap);
 		printf("\n");
 	}
